install_sct

```diff
@@ -118,7 +120,8 @@ function finish() {
     echo ""
   else
     print error "Installation failed!\n
-Please copy the output of this Terminal (starting with the command install_sct) and upload it as a .txt attachment in a new topic on SCT's forum:\n
+Please find the file $(pwd)/install_sct_log.txt and \n
+upload it as a .txt attachment in a new topic on SCT's forum:\n
```

Comment:
Is this how we want it to look?

![Screenshot from 2021-11-17 14-01-05](https://user-images.githubusercontent.com/16181459/142265438-83e3fe5c-41c4-4f7d-8a50-3599e27b1435.png)

I'm guessing `print` preserves newlines, so we're doubling up on them here.


```suggestion
Please find the file $(pwd)/install_sct_log.txt, 
then upload it as a .txt attachment in a new topic on SCT's forum:
```

How's this? 

![Screenshot from 2021-11-17 14-05-32](https://user-images.githubusercontent.com/16181459/142266020-3dd400e2-5c32-41ae-8210-44ff2e5fe18b.png)

Reply:
Yeah that seems good. Thanks for testing that. I thought the extra newlines were a bit weird in the old version but I didn't want to think about futzing with them...